.github/workflows/ci-dependabot.yml

```diff
@@ -33,8 +33,10 @@ jobs:
         run: |
           echo "FILE_NAME=.changelog/${{ github.event.pull_request.number }}.internal.md" >> $GITHUB_OUTPUT
       - name: Create Change Log file
+        env:
+          TITLE: ${{ github.event.pull_request.title }}
```

Comment:
I would add a comment linking to https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-an-intermediate-environment-variable
or rename so that it implies this is done to prevent injection